Maskbook/Scene/MainTab/MainTabBarController.swift

```diff
@@ -116,16 +116,22 @@ extension MainTabBarController {
         tabControllerMap[tab]
     }
 
-    func replace(tab: Tab, with controller: UIViewController, animated: Bool = false) {
+    func replace(tab: Tab, with controller: UIViewController, animated: Bool = false, selected: Bool = false) {
         guard let navigationController = tabControllerMap[tab] else {
             return
         }
 
         navigationController.setViewControllers([controller], animated: animated)
-        selectedIndex = selectedIndex
 
-        if tab.rawValue == selectedIndex, self.tabBar.isHidden {
-            self.tabBar.isHidden = false
+        if selected {
+            // replace navigation stack of tab and select tab
+            selectedIndex = tab.rawValue
```

Comment:
Without `self.tabBar.isHidden = false`, It will hide tabbar after replace?

Reply:
no